frontend/src/MenuPaw.tsx

```diff
@@ -1,14 +1,24 @@
 import menuPaw from "../src/assets/menuPaw.svg";
 import {Link} from "react-router-dom";
 
-export default function MenuPaw() {
+type MenuPawProps = {
+    editMode : boolean,
+    toggleEditMode: () => void;
```

Comment:
```suggestion
    readonly editMode : boolean,
    readonly toggleEditMode: () => void;
```
Das kannst du an alle deiner Props schreiben um die Code smells los zu werden.